src/Raven.Server/Rachis/Candidate.cs

```diff
@@ -135,8 +136,16 @@ private void Run()
 
                         if (realElectionsCount >= majority)
                         {
+                            var connections = new Dictionary<string, RemoteConnection>();
+                            foreach (var candidateAmbassador in _voters)
+                            {
+                                candidateAmbassador.ElectionWon = true;
+                                connections[candidateAmbassador.Tag] = candidateAmbassador.Connection;
```

Comment:
What happens if this is a broken connection at this time? 
Or it was broken immediately after this line?

Reply:
So the follower ambassador should fail to communicate and try to reconnect with a fresh stream.